app/uk/gov/hmrc/helptosavereminder/controllers/EmailCallbackController.scala

```diff
@@ -58,12 +58,13 @@ class EmailCallbackController @Inject()(
                   Future.successful(Ok(s"Error deleting the hts schedule by callBackReference = $callBackReference"))
                 }
                 case Right(()) => {
-                  val path = routes.HtsUserUpdateController.deleteHtsUser().url
+                  val path = routes.EmailCallbackController.handleCallBack(callBackReference).url
                   auditor.sendEvent(
                     HtsReminderUserDeletedEvent(
-                      HtsReminderUserDeleted(htsUserSchedule.nino.toString, Json.toJson(htsUserSchedule)),
+                      HtsReminderUserDeleted(Json.toJson(htsUserSchedule.nino, htsUserSchedule.email)),
```

Comment:
As with the frontend, the event class name has "User" disease & we shouldn't pass JSON into it for lack of type safety

Reply:
done